src/odemis/acq/drift/__init__.py

```diff
@@ -44,23 +44,32 @@ class AnchoredEstimator(object):
     the global acquire, such as at the beginning of a line), and call .estimate()
     to measure the drift.
     """
-    def __init__(self, scanner, detector, region, dwell_time):
+
+    def __init__(self, scanner, detector, region, dwell_time, max_pixels=MAX_PIXELS, follow_drift=True):
         """
         scanner (Emitter)
         detector (Detector)
         region (4 floats)
         dwell_time (float)
+        #TOdo
+        max_pixels (int): the maximum number of pixels that the anchor region will use. If the region
+          acquired is large, then the scale is adjusted to maximum pixels
```

Comment:
"then the scale is adjusted to maximum pixels" -> "then the scanner scale is adjusted so that the anchor region contains at most max_pixels pixels."